net/unbound/files/unbound.sh

```diff
@@ -1067,28 +1065,33 @@ unbound_start() {
 
     unbound_control
   fi
+
```

Comment:
The "method in the madness" would presumably be that it takes care of the preparatory steps required for running unbound jailed (in UNBOUND_VARDIR) ? For /tmp/resolv.conf which lives outside the jail and is only managed by the init routines, not by the unbound daemon, no such requirement exists.

Writing the resolv.conf in unbound_mkdir() breaks the effort to write resolv.conf at the latest possible in order to minimise the chance that a process finds itself firing domain requests at a resolver that is not yet running. That has some value.